agent/xds/builtinextensions/ratelimit/copied.go

```diff
@@ -0,0 +1,59 @@
+package ratelimit
```

Comment:
We've reached the magical number 3 where we should probably just move these into xdscommon.